.rubocop.yml

```diff
@@ -2,3 +2,11 @@
 # alphabetically
 inherit_from:
   - https://raw.githubusercontent.com/hanami/devtools/master/.rubocop.yml
+
+AllCops:
+  TargetRubyVersion: 2.3
+  Exclude:
+    - 'spec/support/fixtures/foo'
+
+Style/SpecialGlobalVars:
```

Comment:
Hm so it looks like the alternative requires adding [`require "English"](https://github.com/rubocop-hq/rubocop/issues/1747) to the top of files that use the special global vars. That's pretty weird so I'm not sure which we'd prefer. We might want to move this to `hanami/devtools`'s `.rubocop.yml` but for now I think it's OK, since it might be the only place this is an issue.